pais.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Jan  3 14:05:31 2020
+
+@author: paisl
+"""
+
+import pandas as pd
+
+
+def feature_engineer(df:pd.DataFrame) -> pd.DataFrame:
+    
+    '''
+    Feature engineering for:
+        Electrical
+        Heating
+        HeatingQC
+        KitchenQual
+        CentralAir
+    '''
+    # Isolate my features
+    featurelist = ['Electrical',
+                   'Heating',
+                   'HeatingQC',
+                   'KitchenQual',
+                   'CentralAir']
+    
+    curr_df = df[featurelist]
+    
+    #Fill 1 missing Electrical value with the mode 
+    mode = 'SBrkr'

```

Comment:
So, this works, but perhaps next time find the mode algorithmically. That way if we're looking at it later and decide that we shouldve used mean or something, we can change it easily.